src/edu/csus/ecs/pc2/ui/ResultsComparePane.java

```diff
@@ -0,0 +1,461 @@
+        centerPane.add(pc2SourcePanel);
+
+        exportDirectoryLabel = new JLabel("PC2 Results Directory");
+        exportDirectoryLabel.setToolTipText("");
+        pc2SourcePanel.add(exportDirectoryLabel);
+
+        pc2ResultsDirectoryTextField = new JTextField();
+        pc2ResultsDirectoryTextField.addKeyListener(new KeyAdapter() {
+            @Override
+            public void keyTyped(KeyEvent e) {
+
+                if (e.getKeyCode() == java.awt.event.KeyEvent.VK_ENTER) {
+                    updateUserSetting(ClientSettings.PC2_RESULTS_DIR, pc2ResultsDirectoryTextField.getText());
+                }
+
+            }
+        });
+        pc2ResultsDirectoryTextField.setToolTipText("export pc2 results files directory ");
+        pc2ResultsDirectoryTextField.setColumns(40);
+        pc2SourcePanel.add(pc2ResultsDirectoryTextField);
+
+        JButton selectExportDirectoryButton = new JButton("Select...");
+        pc2SourcePanel.add(selectExportDirectoryButton);
+        selectExportDirectoryButton.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                selectUpdateExportDirectory(pc2ResultsDirectoryTextField);
+            }
+        });
+        selectExportDirectoryButton.setToolTipText("Select export Directory");
+        
+        JPanel summaryResultsPane = new JPanel();
+        summaryResultsPane.setPreferredSize(new Dimension(800, 300));
+        summaryResultsPane.setMinimumSize(new Dimension(300, 300));
+        summaryResultsPane.setBorder(new TitledBorder(null, "Compare Results Summary", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+        centerPane.add(summaryResultsPane);
+        summaryResultsPane.setLayout(new BorderLayout(0, 0));
+        
+        JPanel resCenterPane = new JPanel();
+        summaryResultsPane.add(resCenterPane, BorderLayout.CENTER);
+        resCenterPane.setLayout(new BorderLayout(0, 0));
+        textArea.setFont(new Font("Courier New", Font.PLAIN, 13));
+        
+        
+//        resCenterPane.add(textArea);
+        
+        JScrollPane scrollPane = new JScrollPane(textArea);
+        resCenterPane.add(scrollPane, BorderLayout.CENTER);
+        
+        JPanel resButtonPane = new JPanel();
+        FlowLayout flowLayout = (FlowLayout) resButtonPane.getLayout();
+        flowLayout.setHgap(45);
+        summaryResultsPane.add(resButtonPane, BorderLayout.SOUTH);
+        
+        JButton viewReportButton = new JButton("View Report");
+        viewReportButton.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                viewCompareResultsFiles();
+            }
+        });
+        resButtonPane.add(viewReportButton);
+        
+        
+        showDetailsCheckbox = new JCheckBox("Include details in comparison");
+        resButtonPane.add(showDetailsCheckbox);
+        showDetailsCheckbox.setToolTipText("Show full detils in comparison");
```

Comment:
The word "details" is misspelled in the above string.  This is not a SHOW STOPPER, but I sure hate to expose misspellings to users -- it reflects poorly on the quality of our product.

Reply:
fixe typo